### DISCUSSION TITLE
How can I get public sponsor badge?

### DISCUSSION THREAD

**Discussion by thundra1125** [community/community#19489] [Other Feature Feedback, Questions, & Ideas]
I want to make sponsor badgemmm

**Comment by melvincwng** [comment]
@Webmaster1125, to get the Public Sponsor badge, simply click on a developer's profile and sponsor him/her! There should be a 'Sponsor' button for those developers who have enabled this function.

You can also sponsor projects as well.

Hope this clarifies.

**Reply by melvincwng** [reply]
@Webmaster1125, for example, this is my [sponsors link](https://github.com/sponsors/melvincwng) for those who wish to sponsor me & support my software development journey.

You can find whichever creators, projects, or developers that you follow and sponsor them.

This will unlock the badge for you. But do take note, once you stop sponsoring them, the badge will also disappear.

If this reply has been useful, please 'Mark as answer'.

Thank you and have a great day ahead 😁

**Comment by thundra1125** [comment]
Thanks very much

**Reply by melvincwng** [reply]
@Webmaster1125, pleasure to have helped you :)

Please kindly 'Mark as answer' so that the others know this issue has been addressed 👍🏻

**Comment by RonaldGuilhermePDS** [comment]
https://github.com/Schweinepriester/github-profile-achievements
